src/UI/WB.UI.Designer/Controllers/Api/Designer/CommentsController.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace WB.UI.Designer.Controllers.Api.Designer
 {
-    [Authorize]
+    [AuthorizeOrAnonymousQuestionnaire]

```

Comment:
not sure it suppose to be here. It looks like that we need Authorize here

Reply:
Discussed